javalanglib/src/main/scala/java/lang/Integer.scala

```diff
@@ -103,6 +113,21 @@ object Integer {
     ((t2 + (t2 >> 4) & 0xF0F0F0F) * 0x1010101) >> 24
   }
 
+  def highestOneBit(i: Int): Int = {
+    // ported from https://github.com/gwtproject/gwt/blob/master/user/super/com/google/gwt/emul/java/lang/Integer.java
+    if (i < 0) {
+      Integer.MIN_VALUE
+    } else if (i == 0) {
+      0
+    } else {
+      var rtn: Int = 0x40000000
```

Comment:
Consider writing `var rtn = (1 << 31)`. It is clearer and will be constant folded anyway (how many zeros were there at the end ;-))